subsys/bluetooth/controller/ll_sw/ull_filter.c

```diff
@@ -176,17 +199,15 @@ uint8_t ll_wl_clear(void)
 #endif /* CONFIG_BT_OBSERVER */
 
 #if defined(CONFIG_BT_CTLR_PRIVACY)
-	wl_clear();
+	al_clear();
 #else
-	filter_clear(&wl_filter);
+	filter_clear(&hw_al);
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
-	wl_anon = 0U;
-
 	return 0;
 }
 
-uint8_t ll_wl_add(bt_addr_le_t *addr)
+uint8_t ll_al_add(bt_addr_le_t *addr)
```

Comment:
Possibly outside the scope of this PR, but I suppose that we can apply `const` to some of the accept list functions as well.

Reply:
done